clarity-ext-scripts/install-req.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
-# Changes that were made on the server for the installation of
+# Changes that must occur once on the server for the installation of
 # clarity-ext-scripts
 
-/opt/gls/clarity/miniconda3/bin/conda create -n clarity-ext python=2.
+# This script should be run as the user `glsai`
+set -e
+
+if [ ! -d /opt/gls/clarity/users/glsai/.conda/envs/clarity-ext ]; then
+    /opt/gls/clarity/miniconda3/bin/conda create -n clarity-ext python=2
+fi
+
+mkdir -p ~/bin
+cp ./deployment/clarity-ext ~/bin/
+
+mkdir -p ~/.config/clarity-ext/
+cp ./deployment/clarity-ext.demo.config ~/.config/clarity-ext/clarity-ext.config
```

Comment:
Maybe we want to check if the file exists or not before overwriting it? I'm thinking that it could otherwise cause unwanted extra work resetting the proper config values on each deployment.